src/exec.sh

```diff
@@ -22,14 +22,16 @@ set -e
 set -u
 
 echo '-INPUT VARIABLES---------------'
-echo $1
-echo $2
-echo $3
+echo build-index: $1
+echo tarball: $2
+echo entry-point: $3
+echo timestamp: $4
 echo '-------------------------------'
 
-TARBALL=$1
-ENTRY_POINT=$2
-TIMESTAMP=$3
+BUILD_INDEX=$1
+TARBALL=$2
+ENTRY_POINT=$3
+TIMESTAMP=$4
```

Comment:
It might be more maintainable, even if it's just debugging, to wait until you give them names to echo them:

```bash
BUILD_INDEX=$1
TARBALL=$2
ENTRY_POINT=$3
TIMESTAMP=$4

echo '-INPUT VARIABLES---------------'
echo build-index: $BUILD_INDEX
echo tarball: $TARBALL
echo entry-point: $ENTRY_POINT
echo timestamp: $TIMESTAMP
echo '-------------------------------'
```